test/acceptance/features/steps/steps.py

```diff
@@ -293,8 +330,8 @@ def create_cr(context, cr_name):
     openshift = Openshift()
     yaml = context.text
     output = openshift.oc_apply(yaml)
-    result = re.search(rf'.*{cr_name}.*(created|unchanged)', output)
-    result | should_not.be_none.desc("CRD {cr_name} Created")
+    result = re.search(rf'.*{cr_name}.*(created|unchanged|configured)', output)
```

Comment:
I think variable used here is {crd_name} not {cr_name}.

Reply:
Variable used is cr_name. Please recheck